ntoskrnl/cc/copy.c

```diff
@@ -438,17 +439,24 @@ CcCanIWrite (
     }
 
 #if DBG
-    DPRINT1("Actively deferring write for: %p\n", FileObject);
+    DPRINT1("Actively deferring write for: %p \"%wZ\"\n", FileObject, &FileObject->FileName);
     DPRINT1("Because:\n");
```

Comment:
Example output, copying a 400 MB file with 96 MB RAM:
```
(/ntoskrnl/cc/copy.c:442) Actively deferring write for: FFAA7A90 "\__Windows_XP_IE6.exe"
(/ntoskrnl/cc/copy.c:443) Because:
(/ntoskrnl/cc/copy.c:445)     First try while already throttling
(/ntoskrnl/cc/copy.c:450)     Available pages are below throttle top: 98 <= 1c2

(/ntoskrnl/cc/copy.c:442) Actively deferring write for: FFAA7A90 "\__Windows_XP_IE6.exe"
(/ntoskrnl/cc/copy.c:443) Because:
(/ntoskrnl/cc/copy.c:445)     First try while already throttling
(/ntoskrnl/cc/copy.c:447)     There are too many cache dirty pages: 23c0 + 10 >= 23cb
(/ntoskrnl/cc/copy.c:450)     Available pages are below throttle top: 77 <= 1c2
```

Hum, I need to fix some conditions some more...